src/EnergyPlus/UnitarySystem.cc

```diff
@@ -13973,6 +13973,7 @@ namespace UnitarySystems {
                                     };
                                     General::SolveRoot(state, HumRatAcc, MaxIte, SolFla, SpeedRatio, f, 0.0, 1.0);
                                 }
+                                PartLoadFrac = SpeedRatio;
```

Comment:
The `SpeedRatio` is saved in a local variable which is then assigned to `this->m_CoolingPartLoadFrac`.